srml/staking/src/lib.rs

```diff
@@ -599,16 +587,9 @@ decl_module! {
 						<KtonPool<T>>::mutate(|k| *k -= unbond_kton);
 
 						*active_kton -= unbond_kton;
-						let expired_locks_kton = T::Kton::update_locks(
-							stash,
-							Some(CompositeLock::Unbonding(Lock {
-								amount: unbond_kton,
-								at,
-								reasons: WithdrawReasons::all(),
-							})),
-						);
+						(*kton_detail_lock).unlocking.push(UnlockChunk { value: unbond_kton, until: at });
 						// TODO: check underflow?
-						*total_kton -= expired_locks_kton;
+						*total_kton -= unbond_kton;
```

Comment:
`total_kton - active_kton = unbonding_kton`

Does it means no Kton in unbonding? 

And `total_kton` always equals to `active_kton` now:
	- both of them are increased in `bond_helper_in_kton` at the same time
	- both of them are decreased in `unbond` at the same time

Reply:
I guess total_kton can be removed now. It is now related to slash now, but I think slash should use active_kton/active_ring instead.